aspirateur/zam_aspirateur/amendements/parser.py

```diff
@@ -96,28 +96,51 @@ def parse_bool(text: str) -> bool:
 
 SUBDIV_RE = re.compile(
     r"""^
-        (?:art\.\sadd\.\s(?P<pos>(avant|après))\s)?  # position
-        Article\s
-        (?P<num>\d+|1er)
+        (?:
+            (
+                art\.\sadd\.
+                |
+                Article\(s\)\sadditionnel\(s\)
+            )
+            \s
+            (?P<pos>(avant|après))
+            \s
+        )?  # position
+        article\s
+        (?P<num>\d+|1er|premier)
         (?:\s(?P<mult>\w+))?        # bis, ter, etc.
         (?:\s.*)?                   # junk
         $
     """,
-    re.VERBOSE,
+    (re.VERBOSE | re.IGNORECASE),
 )
 
 
+TITRE_RE = re.compile(r"Titre (?P<num>\w+)(?: .*)?", re.IGNORECASE)
+
+
 def _parse_subdiv(libelle: str) -> Tuple[str, str, str, str]:
     if libelle == "":
         return ("", "", "", "")
-    if libelle.startswith("ANNEXE "):
-        start = len("ANNEXE ")
-        return ("annexe", libelle[start:], "", "")
+
+    if libelle == "Intitulé du projet de loi":
+        return ("titre", "", "", "")
+
+    mo = TITRE_RE.match(libelle)
+    if mo is not None:
+        return "section", mo.group("num"), "", ""
```

Comment:
Adding parenthesis for consistency or… remove parenthesis everywhere?

Reply:
What does black say? 😀

Reply:
[🦊](https://www.youtube.com/watch?v=jofNR_WkoCE)